src/electron/index.ts

```diff
@@ -447,6 +450,9 @@ function main() {
   // Disconnects from the current server, if any.
   promiseIpc.on('stop-proxying', stopVpn);
 
+  // Install backend services (for Linux only)
+  promiseIpc.on('install-outline-services', RoutingDaemon.installRoutingServices);
```

Comment:
Can you use `ipcRenderer.invoke/handle` instead of `promiseIpc`?
https://www.electronjs.org/docs/latest/api/ipc-renderer#ipcrendererinvokechannel-args

I think promiseIpc forces us to have nodeIntegration, which I'd like to remove.

Reply:
Good idea. I was also thinking of that. Maybe previously our electron version is too old to use `ipcRenderer.invoke`.

Reply:
I used `ipcRenderer.invoke` for `install-outline-services` IPC call. We can migrate other IPC calls as well (like `start-proxy`), let me do it in another PR.